database/schema/mysql-schema.dump

```diff
@@ -1295,5 +1295,3 @@ UNLOCK TABLES;
 /*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
 
 -- Dump completed on 2021-04-26 23:12:04
-
-ALTER TABLE `users` ENCRYPTION='Y';
```

Comment:
The associated instructions should be removed from the readme.

Reply:
Updated.